### ISSUE TITLE
Final steps before publication of 1.0 (tone only)

### ISSUE THREAD

**Issue by samreid** [phetsims/waves-intro#6]
At today's Waves Intro sonification meeting, @kathy-phet said she would like to publish 1.0 without sonification.  Here are some steps I can think of that need to be done before publication.

- [x] Finalize design?
- [x] Make sure Wave Interference 2.0 issues are in good shape, see https://github.com/phetsims/wave-interference/issues/372.  UPDATE: There are no NumberControls in Waves Intro, so those issues don't need to be resolved before going to publication.  But I wonder if taking SHAs at same time would be advantageous to make the sims easier to maintain?
- [x] https://github.com/phetsims/wave-interference/issues/393 suppress oscillator during step
- [x] Interviews
- [x] Pixel polishing
- [x] Disable tab navigation or make sure it is OK fo 1.0, see https://github.com/phetsims/wave-interference/issues/412
- [x] Code review, see https://github.com/phetsims/wave-interference/issues/414 and https://github.com/phetsims/wave-interference/issues/373#issuecomment-488381913
- [x] Dev testing, see https://github.com/phetsims/QA/issues/389
- [ ] RC Testing

If we don't get SHAs until the later steps, we will need to make sure master remains safe while working on sonification.

Assigning to @arouinfar to see if anything is left for design, and to move forward with the first few checkboxes.

**Comment by kathy-phet** [comment]
To clarify ... Finalize design means add a play sound checkbox to sound screen.

Sent from my iPhone

On Apr 23, 2019, at 1:22 PM, Sam Reid <notifications@github.com<mailto:notifications@github.com>> wrote:


At today's Waves Intro sonification meeting, @kathy-phet<https://github.com/kathy-phet> said she would like to publish 1.0 without sonification. Here are some steps I can think of that need to be done before publication.

  *    Finalize design?
  *    Interviews
  *    Pixel polishing
  *    Dev testing
  *    Code review
  *    RC Testing

If we don't get SHAs until the later steps, we will need to make sure master remains safe while working on sonification.

Assigning to @arouinfar<https://github.com/arouinfar> to see if anything is left for design, and to move forward with the first few checkboxes.

**Comment by arouinfar** [comment]
@samreid that list looks good to me. The pixels were polished during Wave Interference design, so I don't have anything further to suggest. I'll go ahead and check that off.

> To clarify ... Finalize design means add a play sound checkbox to sound screen.

I would agree @kathy-phet. Looks like @samreid already has the Play Sound checkbox implemented, so I think we can proceed with scheduling interviews soon.

**Comment by arouinfar** [comment]
Looks like https://github.com/phetsims/wave-interference/issues/373 is still in progress. @samreid can you reassign to me once that's wrapped up, and then I'll schedule interviews.

**Comment by samreid** [comment]
Also blocked by https://github.com/phetsims/wave-interference/issues/384 and https://github.com/phetsims/wave-interference/issues/386.

UPDATE: Now just blocked by phetsims/wave-interference#386

**Comment by samreid** [comment]
At today's meeting, @kathy-phet confirmed we would like to publish this without full sonification, and to make sure it is complete before school starts in the fall.  I'll increase priority and see what else remains to be done here.

**Comment by samreid** [comment]
I published a dev version which could be used for interviews:
https://phet-dev.colorado.edu/html/waves-intro/1.0.0-dev.7/phet/waves-intro_en_phet.html

Please sanity check it first, since it is cut from master.

**Comment by arouinfar** [comment]
@samreid dev.7 looks good. I've already done a round of interviews with dev.6 (play tone only), so I'll wait for #7 before interviewing again.

**Comment by samreid** [comment]
I published a dev version here: https://phet-dev.colorado.edu/html/waves-intro/1.0.0-dev.8/phet/waves-intro_en_phet.html

UDPATE: Dev test QA issue created here: https://github.com/phetsims/QA/issues/370
UPDATE: Significant changes for  https://github.com/phetsims/wave-interference/issues/414 led me to close the dev test. I'll wait for more progress on code review before beginning dev test.
UPDATE: New dev test underway in https://github.com/phetsims/QA/issues/389

**Comment by samreid** [comment]
Testing is underway, and I opened https://github.com/phetsims/wave-interference/issues/441 to continue.  Closing.
